src/htmlexporter.cpp

```diff
@@ -327,47 +327,10 @@ void HTMLExporter::exportBasket(BasketScene *basket, bool isSubBasket)
     stream << QString(
         "  </div>\n"
         "  <p class=\"credits\">%1</p>\n").arg(
-            i18n("Made with <a href=\"http://basket.kde.org/\">%1</a> %2, a KDE tool to take notes and keep information at hand.",
+            i18n("Made with <a href=\"http://launchpad.net/basket\">%1</a> %2, a kdelibs-based tool to take notes and keep information at hand.",
```

Comment:
I suggest removing commit 4bf42c9 from the pull request.
* "kdelibs" sounds too technical
* homepage has been already updated in commit a12a8f8c71b06c4ce9f6083fdc7f6f041b0a887e (I'll cherrypick KF5 commits containing relevant fixes)

Thanks